pkg/distributor/otel.go

```diff
@@ -554,13 +554,15 @@ func otelMetricsToSeriesAndMetadata(
 	logger log.Logger,
 ) ([]mimirpb.PreallocTimeseries, []*mimirpb.MetricMetadata, int, error) {
 	settings := prometheusremotewrite.Settings{
-		AddMetricSuffixes:                 opts.addSuffixes,
-		PromoteResourceAttributes:         prometheusremotewrite.NewPromoteResourceAttributes(config.OTLPConfig{PromoteResourceAttributes: opts.promoteResourceAttributes}),
-		KeepIdentifyingResourceAttributes: opts.keepIdentifyingResourceAttributes,
-		ConvertHistogramsToNHCB:           opts.convertHistogramsToNHCB,
-		PromoteScopeMetadata:              opts.promoteScopeMetadata,
-		AllowDeltaTemporality:             opts.allowDeltaTemporality,
-		AllowUTF8:                         opts.allowUTF8,
+		AddMetricSuffixes:                    opts.addSuffixes,
+		PromoteResourceAttributes:            prometheusremotewrite.NewPromoteResourceAttributes(config.OTLPConfig{PromoteResourceAttributes: opts.promoteResourceAttributes}),
+		KeepIdentifyingResourceAttributes:    opts.keepIdentifyingResourceAttributes,
+		ConvertHistogramsToNHCB:              opts.convertHistogramsToNHCB,
+		PromoteScopeMetadata:                 opts.promoteScopeMetadata,
+		AllowDeltaTemporality:                opts.allowDeltaTemporality,
+		AllowUTF8:                            opts.allowUTF8,
+		LabelNameUnderscoreSanitization:      true,
```

Comment:
my understanding is that hardcoding these two options is a noop and preserves the previous behaviour. Is that right?